j-db-client/prisma/schema.prisma

```diff
@@ -120,6 +120,8 @@ model Post {
   threads      Thread[]
   postTopics   PostTopic[]
   postComments PostComment[]
+
+  @@index(fields: [title, body], name: "title_body_index")
```

Comment:
Great effort here but after having a brief look I don't think this is quite going to work in the way that we want. We'll have more concrete suggestions on this soon after taking a closer look

Reply:
I think I see now, there are two index types for text search in PostgreSQL: GiST and GIN, and... prisma doesn't support them.

Reply:
Yeah it seems so. I'll talk with the folks at Prisma and see if maybe we can help to get support into a coming release